feature-group-definitions/dialog.yml

```diff
@@ -1,4 +1,5 @@
 name: "<dialog>"
+description: The `<dialog>` HTML element represents a modal or non-modal dialog box.
```

Comment:
How about an example or two?

```suggestion
description: The `<dialog>` HTML element represents a modal or non-modal dialog box, such as a confirmation prompt or inspector widget.
```

Reply:
Not sure what an "inspector widget" is. Maybe:

```suggestion
description: The `<dialog>` HTML element represents a modal or non-modal dialog box, such as a confirmation prompt or a subwindow used to enter data.
```

Reply:
Not a Mac user, eh? 😄 But yes, this works!